### ISSUE TITLE
Dynamic header generation for array items

### ISSUE THREAD

**Issue by ErikKoinberg** [bhch/django-jsonform#149]
It would be nice to have an option to generate dynamic headers when adding new items to an array. Perhaps a property called objectName or something like that is needed. For an object name "Book", it would be cool to get subheaders for the items like Book 1, Book 2 etc.

**Comment by bhch** [comment]
Any ideas how this can be implemented?

Only way I can think of is to execute a JS function every time a new item is added and that function will return a new header?

**Comment by ErikKoinberg** [comment]
Yes, I agree. The function will have to look at the schema for what the object type name is as well as how many there currently are. With reordering or even duplications, this gets even trickier.

**Comment by ErikKoinberg** [comment]
Yes, I agree. The function will have to look at the schema for what the object type name is as well as how many there currently are. With reordering or even duplications, this gets even trickier.
